plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -126,6 +126,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         switch (connectorBehavior) {
             case SUPPORTS_DELETE:
             case SUPPORTS_TRUNCATE:
+                return true;
```

Comment:
Was it inter-related for any reason ?

Reply:
Logical mistake in https://github.com/trinodb/trino/pull/13334/files#diff-6d69520925faa8f876d78d002c7ad94202dfc4a81a9758a5bc3f2ba93b40e023R128

One more reason in favour of @findepi's idea to keep only related enums grouped.

Reply:
I think that was just a mistake, there should be no relation.
Thanks for fixing